### DISCUSSION TITLE
无法连接节点但可ping通无拦截frp

### DISCUSSION THREAD

**Discussion by LiplusCu** [natfrp/wiki#283] [使用问题] [answered]
不知为何在别处使用sakurafrp均无问题 也可以ping通 有的网络使用就会提示 登录节点失败, 请检查网络连接: dial tcp 121.62.17.41:7000: i/o timeout
但是使用其他frp服务并没有问题

**Comment by LiplusCu** [comment]
已知道问题 7000端口不通 话说为啥 有没有什么解决办法嘞

**Comment by fakeboboliu** [comment, marked as answer]
通常原因为特定组织网络（如校园网）的网管为了网络安全，阻止了内网穿透类软件的使用。

为了用户的人身安全，我们建议在此情况时尽量不要使用，人在屋檐下。

如果您确认自己的行为不会造成安全隐患，且再次阅读我们的服务条款和隐私协议，可以使用启动器的 “使能 TLS” 开关或命令行参数 `--natfrp_tls`，通常可以绕过此问题。

**Reply by wjcper2008** [reply]
你好，之前还好用的，也启动了--natfrp 这个命令，但是，这几天突然用不了了，还有其他的办法吗？
